source/agora/consensus/state/ValidatorSet.d

```diff
@@ -805,14 +759,35 @@ public class ValidatorSet
 
         return utxos;
     }
+
+    /***************************************************************************
+
+        Calculate the lowest height for an enrollemnt for the given height
+
+        Params:
+            height = block height
+
+        Returns:
+            min height for active enrollment at height given
```

Comment:
Since the function is so trivial, you can use:
```D
Returns:
    Calculate the lowest height an Enrollment had to appear at for it to be a validator at `height`
```

Directly (so `Returns: description` without `Params` or the extra title).

Reply:
Done